src/components/ArbeidsgiverPeriodeTabell.tsx

```diff
@@ -195,13 +194,13 @@ class ArbeidsgiverPeriodeTabell extends Component<Props, State> {
               <div className="arbeidsgiver-periode-tabell--header">
                 <div className="arbeidsgiver-periode-tabell--info-gruppe">
                   {
-                    person &&
+                    sak &&
                     <>
                       <div className="arbeidsgiver-periode-tabell--person-nummer">
-                        Fødselsnummer: {identityNumberSeparation(person?.identitetsnummer ?? '')}
+                        Fødselsnummer: {identityNumberSeparation(sak?.arbeidsgiver.identitetsnummer ?? '')}
```

Comment:
Trodde du brukte språkfil?

Reply:
Kan jobbe på å sette det på nå, men mangler design for hvordan man skal endre språk